README.md

```diff
@@ -31,4 +59,155 @@ from `package.json` and `package-lock.json`
     }
     ```
 
-1. Run `npm install` to download the library and include it in `package-lock.json`
\ No newline at end of file
+1. Run `npm install` to download the library and include it in `package-lock.json`
+
+
+# API
+
+## `BraveAlerter` class
+
+The main class of this library. It is used to send single alerts or to start alert sessions with the responders.
+
+### constructor(getAlertSession, getAlertSessionByPhoneNumber, alertSessionChangedCallback, asksIncidentDetails, getReturnMessage)
+
+**getAlertSession (async function(sessionId)):** function that returns the AlertSession object with the given sessionId
+
+**getAlertSessionByPhoneNumber (async function(toPhoneNumber)):** function that returns the AlertSession object for the most recent unfinished session with the given phone number
+
+**alertSessionChangedCallback (async function(alertSession)):** function that will be called whenever an alertSession's values change; should be used to update the session in the DB
+
+**asksIncidentDetails (boolean):** `true` if alert sessions should ask for incident details, `false` otherwise
+
+**getReturnMessage (function(fromAlertState, toAlertState)):** function that returns the message to return when there is a transition from `fromAlertState` to `toAlertState`
+
+
+### getRouter()
+
+The BraveAlerter's Express Router contains the route `POST /sms` that can be added to an existing Express app by:
```

Comment:
Suggestion: mount all the routes that BraveAlerter creates on a base route, like `/alert` or something similar, to avoid accidental collisions.

Reply:
Avoiding accidental collisions sounds like a good idea. We can either do that here or just change the suggested code for the caller to be `express.use('/alert', braveAlerter.getRouter())`.

And in either case I'm now thinking that `/twilio` would be a better endpoint since nothing else will ever use this.

Reply:
Oh no, turns out that `POST /alert` is already being used in ODetec:
![image](https://user-images.githubusercontent.com/1490437/94878352-eea9e980-0411-11eb-8559-b2030995431c.png)

Shall we go with `POST /bravealert`?

Reply:
Actually, it looks like this code is only in the integration tests that were used for the performance tests. @mariocimet do you think this use of `/alert` can co-exist with the shared library using `/alert`?

Reply:
yeah I think it can coexist/we can do away with this integration testing code

Reply:
Sweet. I won't worry about this then. Thanks!